pkg/storage/storagebackend/factory/factory.go

```diff
@@ -33,7 +33,7 @@ func Create(c storagebackend.ConfigForResource, newFunc func() runtime.Object) (
 	case storagebackend.StorageTypeETCD2:
 		return nil, nil, fmt.Errorf("%s is no longer a supported storage backend", c.Type)
 	case storagebackend.StorageTypeUnset, storagebackend.StorageTypeETCD3:
-		return newETCD3Storage(c, newFunc)
+		return newS3Storage(c, newFunc)
```

Comment:
Just to be clear, the implementation for the `storage.Interface` should live in the aws-backend repo, eventually you would need to switch to [NewAWSStorage](https://github.com/team-pua/aws-backend/blob/1e20532ca649db3ab6509ddbede19860e99be37a/storage/store.go#L203).